charts/fluentd-1.14.5/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+deprecated: true
+description: DEPRECATED A Helm chart for Verrazzano Fluentd
```

Comment:
Can we ditch the VZ branding?  I'd prefer to keep the descriptions/notes/etc. as unbranded as possible.

Reply:
done